spec/requests/post_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET /posts#index' do
+    it 'returns a success response' do
+      get '/users/45/posts'
+      expect(response).to be_successful
+    end
+
+    it 'renders the index template of users post' do
+      get '/users/45/posts'
+      expect(response).to render_template(:index)
+    end
+
+    it 'includes the correct placeholder text' do
+      get '/users/45/posts'
+      expect(response.body).to include('Here is a list of all the post for a particular user')
+    end
+  end
+
+  describe 'GET posts#show' do
+    it 'returns a successful response' do
+      get '/users/45/posts/45'
+      expect(response).to have_http_status(200)
+    end
+
+    it 'renders the post template' do
+      get '/users/45/posts/45'
+      expect(response).to render_template(:show)
+    end
+
+    it 'includes the correct placeholder text' do
+      get '/users/45/posts/45'
+      expect(response.body).to include('Here is a particular post by a specific user')
+    end
+  end
+end
```

Comment:
Test Result 
![image](https://github.com/oguntayomicheal/Blogs-App/assets/78251927/37e94446-42e1-450f-aea6-c8085ffbfad3)

Please, make sure that the spec file includes tests that ensure the correct view template is rendered 🛠

Reply:
Hi @abdulhamiid, all the test are passing from my end.